### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/zGDQQg).

### CODE REVIEW

File: Controller/ImageController.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function __construct(
         ManagerRegistry $registry,
         $managerName,
         $class,
-        $rootPath = '/',
+        $rootPath,
```
Comment (ElectricMaxxx) [inline Controller/ImageController.php]: uh this one is not really good :-(  (created PR automatically)

**Comment by ElectricMaxxx** [comment]
Dear StyleCI,
your really wanna force me to a BC break? Or do you wanna force me to move the parameters with defaults to the end? (Which is a BC break too)

**Comment by dbu** [comment]
oh well. its a string so technically just having the name without default value would result in null, then we could do `if (null === $rootPath) {$rootPath = '/';}` first thing in the constructor. i guess the other parameters just got appended later.

**Comment by ElectricMaxxx** [comment]
Uhh that looks like JavaScript.

**Comment by dbu** [comment]
yeah i know. the problem is that indeed parameters with defaults that are not at the end of the list of parameters basically make no sense. you will always have to suplly a value for them, as you need to pass the parameters that come after them...

**Comment by dbu** [comment]
see #152

**Pull request closed without merging**